turbinia/e2e/e2e-local.sh

```diff
@@ -1,44 +1,61 @@
 #!/bin/bash
 # This scripts executes a Turbinia end-to-end test against a local 
 # docker-compose Turbinia stack
-# The evidence processed is Chrome Browser History.
+# The evidence processed is a prepared raw disk image.
+
+# Set default return value
+RET=0
 
 echo "Create evidence folder"
 mkdir -p ./evidence
 sudo chmod 777 ./evidence
 
-echo "Download Chrome browser history artifact"
-curl https://raw.githubusercontent.com/obsidianforensics/hindsight/master/tests/fixtures/profiles/60/History > History
-tar -vzcf ./evidence/history.tgz History
-ls -al ./evidence/
+echo "==> Copy test artifacts to /evidence"
+cp ./test_data/artifact_disk.dd ./evidence/
 
-echo "Startup local turbinia docker-compose stack"
+echo "==> Startup local turbinia docker-compose stack"
 docker-compose -f ./docker/local/docker-compose.yml up -d
 
-echo "Sleep for 5s"
+echo "==> Sleep for 5s"
 sleep 5s
 
-echo "Show running instances"
+echo "==> Show running instances"
 docker ps -a
 
-echo "Show container logs"
+echo "==> Show loop device availability in worker"
+docker exec -t turbinia-worker /sbin/losetup -a
+docker exec -t turbinia-worker ls -al /dev/loop*
+
+echo "==> Show evidence volume contents in worker"
+docker exec -t turbinia-worker ls -al /evidence/
+
+echo "==> Show container logs"
 docker logs turbinia-server
 docker logs turbinia-worker
 
-echo "Create  Turbinia request"
-docker exec -t turbinia-server turbiniactl -r 123456789 compresseddirectory -l /evidence/history.tgz 
+echo "==> Create  Turbinia request"
+docker exec -t turbinia-server turbiniactl -r 123456789 rawdisk -l /evidence/artifact_disk.dd
 
-echo "Sleep for 60 seconds to let Turbinia process evidence"
-sleep 60s
+echo "==> Sleep for 150 seconds to let Turbinia process evidence"
+sleep 150s
 
-echo "Display Turbinia request status"
+echo "==> Display Turbinia request status"
```

Comment:
I thought about that but the github action max runtime is 6 hours and that seems too long. The 150 second sleep is enough for our known test data to finish and it makes sure we can spot problems soon instead of having to wait possible 6 hours until a test finishes. Not having a known time before a test finishes can lead to people just ignore those tests and/or not look at the output.